tests/input_validation/test_input_validation.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+from typing import Dict, Any
+from supertokens_python import InputAppInfo, SupertokensConfig, init
+from supertokens_python.recipe import passwordless
+
+
+async def send_text_message(param: passwordless.CreateAndSendCustomTextMessageParameters, _: Dict[str, Any]):
+    print(param)
+
+
+@pytest.mark.asyncio
+async def test_init_validation_passwordless():
```

Comment:
You need to change these tests to also check the message associated with the valueerror